web/static/js/socket.js

```diff
@@ -4,7 +4,7 @@ let socket = new Socket("/socket", { params: { token: window.userToken } })
 
 socket.connect()
 
-let gameChannel = socket.channel("game:lobby", {})
+let gameChannel = socket.channel("game:" + window.server, {})
```

Comment:
fyi JS can use string interpolation since ES6 (uses backticks and `${}`)
```js
`game: ${window.server}`

Reply:
If we switch to typescript then we should, otherwise might as well keep es5 support (unless phoenix is transpiling to es5?).

Reply:
it transpiles with Brunch, yeah